tools/clang/test/CodeGenSPIRV/intrinsics.select.hlsl

```diff
@@ -0,0 +1,181 @@
+// RUN: %dxc -T ps_6_0 -E main -HV 2021
+
+// CHECK: [[v3i0:%\d+]] = OpConstantComposite %v3int %int_0 %int_0 %int_0
+SamplerState gSS1;
```

Comment:
I might be wrong, but those aren't used right? Not sure I'm following the commit message, are those supposed to be tested/working yet?

Reply:
Right, you're correct. Thanks for catching that!

Reply:
Shouldn't we test them instead of remove them? (since the bug was found with those?)